schema/properties_directory/elemental/atomic_radius.json

```diff
@@ -12,7 +12,8 @@
         "name": {
             "enum": [
                 "atomic_radius"
-            ]
+            ],
+            "$comment": "isGenerative:true"
```

Comment:
1. Do we ever explain what `isGenerative` actually is? We should add some documentation for this, perhaps even to the user documentation and refer to here.
2. We don't need to inject `isGenerative:true` into every `name` key, but instead assume that `name` is the default generative per https://github.com/Exabyte-io/web-app/pull/2270/files#r1303651176.
3.

Reply:
1. the flag's purpose/meaning is described in the `README.md`
2. flag removed from all names and included by default.